frontend/test/metabase/scenarios/models/reproductions/19737-data-picker-not-showing-moved-model.cy.spec.js

```diff
@@ -13,30 +13,76 @@ describe.skip("issue 19737", () => {
   it("should show moved model in the data picker without refreshing (metabase#19737)", () => {
     cy.visit("/collection/root");
 
-    openEllipsisMenuFor(modelName);
-    popover()
-      .contains("Move")
+    moveModel(modelName, "My personal collection");
+
+    cy.findByText("Moved model");
```

Comment:
Basically, the first test is unchanged, just extracting some steps to `moveModel`